### ISSUE TITLE
Add support for multiple dispatched plugins

### ISSUE THREAD

**Issue by mlaferrera** [PUNCH-Cyber/stoq#20]

**Comment by mlaferrera** [comment]
Initial support is in 5776f87 

dispatcher yara rules now support multiple comma separated plugins in the metadata:

```
rule test_dispatch_rule_multi
{
    meta:
        plugin = "carver:test_carver,decoder:test_decoder,extractor:test_extractor"
        save = "True"
    strings:
        $test = "multi-test"
    condition:
        any of them
}
```

**Comment by mlaferrera** [comment]
Merged into v0.15.0.
